draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -48,6 +48,8 @@
   private static final float SENSITIVITY = 1f;
   private static final boolean DEFAULT_TOP_VIEW_RESIZE = false;
 
+  private boolean mEnableClickToExpand = true;
```

Comment:
In this project I'm not using "m" naming convention. Can you rename this variable name to enableClickToExpand?